_about/notes/0.8.md

```diff
@@ -8,4 +8,47 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model.
+[Learn more]({{home}}/docs/tasks/traffic-management-v1alpha3/)
+
+- **Envoy V2**. Pilot now uses the V2 Envoy API to program the proxies in the mesh, which increases
+effective scalability and eliminates spurious 404 errors.
+
+- **Constrain Inbound Ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
+
+## Security
+
+- **Introducing Citadel**. We've finally given a name to our security component. What was
+formerly known a Istio-Auth or Istio-CA is now called Citadel.
+
+- **Multicluster Support**. [TBD: What should we say?]
+
+- **Authentication Filter**. [TBD: What should we say about the proxy authn filter?]
```

Comment:
This is somewhat implementation detail. Maybe we should announce Authentication policy only.
**Authentication policy**. introduce authentication policy that can be used to configure service-to-service authentication (mTLS) and end user authentication. This is a recommended way for mTLS enablement (over existing config flag and service annotation).